etl-data-pipeline/src/main/resources/db/routines/provider/sp_provider.sql

```diff
@@ -1,18 +1,19 @@
+use NBS_ODSE
+go;
 Create or Alter PROCEDURE [dbo].[sp_PROVIDER_EVENT] @user_id_list varchar(max)
 as
 BEGIN
 
     --UPDATE ACTIVITY_LOG_DETAIL SET
---START_DATE=DATETIME();
+    --START_DATE=DATETIME();
 
-    DECLARE @RowCount_no INT;
     DECLARE @Proc_Step_no FLOAT = 0;
```

Comment:
We are removing these steps as well?

Reply:
Yeah we want to keep the stored proc simple. Let me remove these too.

Reply:
I will keep them for no for documentation purposes as to modularize the stored proc into different steps. They are just harmless titles on top of each step. If we decide to move away from them, we can remove these and morph into comments.